admin/api_ldap_configuration.go

```diff
@@ -153,7 +153,7 @@ func (a *LDAPConfigurationApiService) DeleteLDAPConfigurationWithParams(ctx cont
 	}
 }
 
-func (r DeleteLDAPConfigurationApiRequest) Execute() (*http.Response, error) {
+func (r DeleteLDAPConfigurationApiRequest) Execute() (*UserSecurity, *http.Response, error) {
```

Comment:
In this scenario, a response body was added. 
This likely coming from @gssbzn parity checks.